crates/network-scanner/src/scanner.rs

```diff
@@ -299,11 +341,13 @@ impl Display for ScanMethod {
 #[derive(Debug, Clone, TypedBuilder, Default)]
 pub struct NetworkScannerParams {
     pub ports: Vec<u16>,
-    pub ping_interval: u64,     // in milliseconds
-    pub ping_timeout: u64,      // in milliseconds
-    pub broadcast_timeout: u64, // in milliseconds
-    pub port_scan_timeout: u64, // in milliseconds
-    pub netbios_timeout: u64,   // in milliseconds
-    pub netbios_interval: u64,  // in milliseconds
-    pub max_wait_time: u64,     // max_wait for entire scan duration in milliseconds, suggested!
+    pub ping_interval: u64,             // in milliseconds
+    pub ping_timeout: u64,              // in milliseconds
+    pub broadcast_timeout: u64,         // in milliseconds
+    pub port_scan_timeout: u64,         // in milliseconds
+    pub netbios_timeout: u64,           // in milliseconds
+    pub netbios_interval: u64,          // in milliseconds
+    pub mdns_meta_query_timeout: u64,   // in milliseconds
+    pub mdns_single_query_timeout: u64, // in milliseconds
+    pub max_wait_time: u64,             // max_wait for entire scan duration in milliseconds, suggested!
```

Comment:
thought: We should probably use the `Duration` type instead. Should be addressed separately.